README.md

```diff
@@ -1,14 +1,20 @@
 
 ## Virtual World where people may move about in a full 3D world.
 
-Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D graphics application, commonly known as a "Viewer" which is used to access these virtual worlds. (not a VR headset) such as the most common one, <a href="https://firestormviewer.org" rel="follow">FirestormOS</a>.  *(the "OS" standing for 'OpenSim', and not 'Operating System')* also it is similar to <a href="https://SecondLife.com" rel="nofollow" >Second Life®</a>.  It allows users to "talk" to other people in-world through nearby chat, IM's, group chats and group IM's and sometimes in voice, depending on how the grid is setup and the permissions set on individual regions. Although that's as much like a game as it gets. There is also the ability to build objects in-world known as primatives or prims in various shapes and sizes to make different types of linked or individual 3D objects. Additionally other 3D items known as "Mesh" (DEA files) can be imported which have been created/edited with a 3D object editor/creator such as the free program "<a href="https://store.steampowered.com/app/365670/Blender/" rel="follow" >Blender</a>".
+Halcyon World is a Virtual World with a full 3D environment,  written in C Sharp (C#), where you may move about with a character known as an avatar in a 3D World accessed with a 3D graphics application, commonly known as a "viewer", which is used to access/login to these virtual worlds (not a VR headset). The most common one currently is the <a href="https://firestormviewer.org" rel="follow">Firestorm viewer for OpenSim</a>.  *(the "OS" stands for 'OpenSim', and not 'Operating System')* also it is similar to <a href="https://SecondLife.com" rel="nofollow" >Second Life™</a>.  It allows users to "talk" to other people in-world through nearby chat, IM's, group chats and group IM's and sometimes in voice, depending on how the grid is setup and the permissions set on individual regions. Including offline IM's, sending the offline IM's to the user's registered email address(which most OpenSim Grids do not have). Although not yet with <a href="https://OpenPGP.org" rel="nofollow">OpenPGP</a> Encryption. That is as much like a game as it gets. There is also the ability to build objects in-world known as primatives or "prims" in various shapes and sizes to make different types of linked or individual 3D objects. Additionally other 3D items known as "Mesh" (DEA files) can be imported which have been created/edited with a 3D object editor/creator such as the free program titled "<a href="https://store.steampowered.com/app/365670/Blender/" rel="follow" >Blender</a>". (Or even exported from a Grid as Mesh/DEA.)
```

Comment:
Please omit the jab `(which most OpenSim Grids do not have)`.

Please omit `Although not yet with <a href="https://OpenPGP.org" rel="nofollow">OpenPGP</a> Encryption.` If such a think should exist, it should be in an issue ticket not here in the README - and even then that's more a client thing than a server thing.

Either rework to be part of the flow or omit `(Or even exported from a Grid as Mesh/DEA.)` Parenthetical insertions are not a good writing style, and wouldn't be proper grammar wrapping a partial sentence like this anyway.

Reply:
Alright, thank you, I'm making those changes... and OpenPGP is actually both a client and server thing, at least when sending mail directly from a server. Users/Clients need a [OpenPGP](https://OpenPGP.org) compatible software, and the server does as well, to encrypt the message with each user's public key. Then the user/client/recipient decrypts the message with their private key.